packages/at_client/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ environment:
   sdk: '>=2.12.0 <3.0.0'
 
 dependencies:
-  path: ^1.8.3
+  path: ^1.8.2
```

Comment:
path should be 1.8.2 and not ^1.8.2

Reply:
I disagree.

Reply:
updated

Reply:
I think I misunderstood the context and mistakenly thought Murali's suggestion was to remove the ^